testkit/src/main/java/org/apache/calcite/test/QuidemTest.java

```diff
@@ -150,6 +151,10 @@ protected void checkRun(String path) throws Exception {
                   && (Boolean) value;
               closer.add(Prepare.THREAD_EXPAND.push(b));
             }
+            if (propertyName.equals("insubquerythreshold")) {
```

Comment:
is this temporary ? surprised code gen part follows naturely and requires no change.

Reply:
No. This can be used in the .iq file to set the insubquerythreshold property.

Reply:
`!set insubquerythreshold 0`

Reply:
@zinking Please review again, Then we can merge.